packages/webkit2gtk_4.rb

```diff
@@ -1,133 +1,180 @@
-require 'package'
-
 class Webkit2gtk_4 < Package
   description 'Web content engine for GTK'
   homepage 'https://webkitgtk.org'
-  @_ver = '2.32.4'
-  version @_ver
+  case ARCH
+  when 'x86_64'
+    version '2.38.1'
+  when 'i686'
+    version '2.38.0-1'
+  when 'aarch64', 'armv7l'
+    version '2.32.4'
+  end
   license 'LGPL-2+ and BSD-2'
   compatibility 'all'
-  source_url "https://webkitgtk.org/releases/webkitgtk-#{@_ver}.tar.xz"
-  source_sha256 '00ce2d3f798d7bc5e9039d9059f0c3c974d51de38c8b716f00e94452a177d3fd'
+  source_url 'https://webkitgtk.org/releases/webkitgtk-2.38.1.tar.xz'
+  source_sha256 '02e195b3fb9e057743b3364ee7f1eec13f71614226849544c07c32a73b8f1848'
 
   binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.32.4_armv7l/webkit2gtk_4-2.32.4-chromeos-armv7l.tpxz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.32.4_armv7l/webkit2gtk_4-2.32.4-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.32.4_i686/webkit2gtk_4-2.32.4-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.32.4_x86_64/webkit2gtk_4-2.32.4-chromeos-x86_64.tpxz'
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.38.0-1_i686/webkit2gtk_4-2.38.0-1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/webkit2gtk_4/2.38.1_x86_64/webkit2gtk_4-2.38.1-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
     aarch64: 'd25a0be821cbf2c710539e685268d47bdcde109ed5a18b2202c132b31b341219',
      armv7l: 'd25a0be821cbf2c710539e685268d47bdcde109ed5a18b2202c132b31b341219',
-       i686: '1a5272472c312165616317481b406085a237f7f5748ab284a826af1faf9c541e',
-     x86_64: '36c5c54bc95ede8c00fb0b35361c4c20213ab6129a2712c10f91ff9c08128942'
+       i686: '5c7acadd5fdf295f2590418c09a87cd9ca305b1af34f982fb5a3e03adc78db88',
+     x86_64: '117945f103fe1af58638451841fff222d5b8c699eb65c05215cce6613f765a65'
```

Comment:
What is wrong with the arm builds?

Reply:
have you checked around to see if maybe fedora, ubuntu or arch linux have a patch for arm32?